go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/databricks/terraform-provider-databricks
 go 1.21
 
 require (
-	github.com/databricks/databricks-sdk-go v0.30.0
+	github.com/databricks/databricks-sdk-go v0.31.0
```

Comment:
i just did a `go get github.com/databricks/databricks-sdk-go` and I suppose the new SDK requires newer versions of the other dependencies?